templates/nginx.conf.j2

```diff
@@ -80,3 +81,12 @@ server {
         deny all;
     }
 }
+
+{% if laravel_https_redirect -%}
```

Comment:
Since laravel_https_redirect is enabled by default, better also check laravel_https_enabled here. Or make it disabled by default.